pkgs/applications/editors/vim/plugins/generated.nix

```diff
@@ -4678,6 +4678,19 @@ final: prev:
     meta.homepage = "https://github.com/rebelot/heirline.nvim/";
   };
 
+  helpview-nvim = buildVimPlugin {
+    pname = "helpview.nvim";
+    version = "2024-08-12";
+    src = fetchFromGitHub {
+      owner = "OXY2DEV";
+      repo = "helpview.nvim";
+      rev = "49398d628d74283c5a389a6ced40bd8d2daaf56c";
+      sha256 = "0b1z3gz1jpqzf71kjpkw9q3fxbjcdykxi65an6npci0acia49ga2";
+      fetchSubmodules = true;
```

Comment:
```suggestion
```

I am not sure we need to fetch the "wiki" section of the repo.

Reply:
Update script automatically fetches all submodules, if any found. This line will just return on the next update.